docs/strategies/timeout.md

```diff
@@ -108,6 +109,19 @@ catch (TimeoutRejectedException)
 ```
 <!-- endSnippet -->
 
+## Defaults
+
+| Property           | Default Value | Description                                                                                                |
+|--------------------|---------------|------------------------------------------------------------------------------------------------------------|
+| `Timeout`          | 30 seconds    | It defines a **static** period within the delegate should complete otherwise it will be cancelled.         |
```

Comment:
```suggestion
| `Timeout`          | 30 seconds    | Defines a **static** period within which the delegate should complete, otherwise it will be cancelled.         |
```